ddpui/models/admin_audit_log.py

```
@@ -0,0 +1,29 @@
+from django.db import models
+
+from ddpui.models.org import Org
+
+
+class AdminAuditLog(models.Model):
+    """Stores admin audit logs for sensitive admin actions."""
+
+    ACTION_CHOICES = [
+        ("organization_updated", "organization_updated"),
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)